android/src/main/java/com/reactnativecommunity/webview/RNCWebViewManager.java

```diff
@@ -1480,6 +1493,14 @@ protected ViewGroup getRootView() {
     public void setProgressChangedFilter(RNCWebView.ProgressChangedFilter filter) {
       progressChangedFilter = filter;
     }
+
+    /**
+     * Set whether or not protected media should be allowed
+     * /!\ Setting this to fault won't revoke permission already granted to the current webview /!\ 
```

Comment:
nit: 
```suggestion
     * /!\ Setting this to false won't revoke permission already granted to the current webview /!\ 
```

Also, this warning might be worth adding to the documentation?

Reply:
Good catch thanks, and sorry for that typo!

I've added that to the documentation, and also I've edited this warning to be more accurate. As a matter of fact, I've performed more tests and tried to toggle the allowsProtectedMedia value and it appears that reloading the page would actually result in a new permission request and thus, take the allowsProtectedMedia value change into account.

Let me know if you think the explanation is clear enough.